doc-changelog/action.yml

```diff
@@ -82,7 +82,7 @@ runs:
     - name: "Install towncrier"
       shell: bash
       run: |
-        python -m pip install --upgrade pip towncrier
```

Comment:
Could we pass this as an input with default value `23.11.0`?

Reply:
Should I do the same for the doc-deploy-changelog action too?

Reply:
Yep please do